websubhub-ballerina/utils.bal

```diff
@@ -20,22 +20,26 @@ import ballerina/http;
 import ballerina/uuid;
 import ballerina/regex;
 
+# Wrapper class used to execute sevice methods.
+isolated class RequestHandler {
+}
+
```

Comment:
I think this is better.

isolated class RequestHandler {
     isolated function callOnUnsubscriptionIntentVerifiedMethod(VerifiedUnsubscription msg, http:Headers headers) {} extern.
}